### ISSUE TITLE
Add enums to data types meta file

### ISSUE THREAD

**Issue by geekygirlsarah** [codethesaurus/codethesaur.us#510]
# Background

Data types are a "concept" in Code Thesaurus that shows mostly simpler data types. I think enums are things in a lot of languages, and could be added to the `/web/thesauruses/_meta/data_types.json` file.

# Implementation

Use the existing file, but add a category for "Enumerations".  From there, add in several concepts that could describe what enums look like in many languages. (For example, you could think of languages you know, or head to [Wikipedia](https://en.wikipedia.org/wiki/Enumerated_type) to see how they show they're done in a variety of languages.) From there, make sure the concepts can adequately describe how they work in various languages in ways that could be compared. Also give them good descriptions.

# Notes

Because the meta files can subsequently affect many language definitions of this type, this may go through a more rigorous review process. We may chat back and forth about ideas on how to work through this. If there's conversations on this pull request, it's more just thinking through the future and less an indication that the implementation is bad. I just want to ensure we don't have to correct things later!

**Comment by piromsurang** [comment]
Could I work on this task?

**Comment by geekygirlsarah** [comment]
Yeah! This is a little bit more complicated, so a pull request/code review may take a bit longer, but I'll assign you.

Let me know if you have any questions. There's also the docs at https://docs.codethesaur.us too!
